pom.xml

```diff
@@ -102,6 +102,15 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-logging-sentry</artifactId>
     </dependency>
+    <dependency>
+      <!-- This builds
+      <groupId>com.redhat.cloud.common</groupId>
+      <artifactId>java</artifactId>
+      -->
+      <version>184c3df8c6</version>
```

Comment:
Do you have plans to automate releases on the common code to make this version more readable?

Reply:
Yes. They are now on JBoss public repo and will eventually be synced to Central.